module/VuFind/sql/mysql.sql

```diff
@@ -409,3 +409,24 @@ CREATE TABLE `access_token` (
   CONSTRAINT `access_token_ibfk_1` FOREIGN KEY (`user_id`) REFERENCES `user` (`id`) ON DELETE CASCADE
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE utf8mb4_unicode_ci;
 /*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Table structure for table `login_token`
+--
+
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8mb4 */;
+CREATE TABLE `login_token` (
```

Comment:
Once things are finalized, we should add equivalent code to the PostgreSQL configuration. I've added a TODO checkbox to the top of the PR to remind us of this.